public/app/core/components/NavBar/utils.ts

```diff
@@ -73,35 +75,61 @@ export const enrichConfigItems = (
   return items;
 };
 
-export const isLinkActive = (pathname: string, link: NavModelItem) => {
-  // strip out any query params
-  const linkPathname = link.url?.split('?')[0];
+export const isMatchOrChildMatch = (itemToCheck: NavModelItem, searchItem?: NavModelItem) => {
+  return Boolean(itemToCheck === searchItem || itemToCheck.children?.some((child) => child === searchItem));
```

Comment:
small nit: don't think this needs the `Boolean()` - both sides of the `||` are already a boolean, no?

Reply:
ahhh - the `children?`

Reply:
@joshhunt there's the optional chaining on children which can make the RHS `undefined`